daemon/checkpoint.go

```diff
@@ -17,7 +17,7 @@ var (
 )
 
 // getCheckpointDir verifies checkpoint directory for create,remove, list options and checks if checkpoint already exists
-func getCheckpointDir(checkDir, checkpointID, ctrName, ctrID, ctrCheckpointDir string, create bool) (string, error) {
+func getCheckpointDir(checkDir, checkpointID, ctrName, ctrCheckpointDir string, create bool) (string, error) {
```

Comment:
Looks like this was used in the past, but its use was removed in https://github.com/moby/moby/commit/e51aec992624fec305176866f3937e0a120cecb8, so looks fine to remove;

- https://github.com/moby/moby/pull/35694